ubcpi/templates/ubcpi.html

```diff
@@ -0,0 +1,290 @@
+            <div style="margin: 0 0 1.5em 0; padding: 1em; background: #f8f8f8;">
+            <img ng-src="{{question_text.image_url}}" id="question-image" alt="{{question_text.image_alt}}" ng-if="question_text.image_position == 'above' && question_text.image_url" />
+
+            <span aria-labelledby="pi-question-h" id="question-text" ng-bind-html="question_text.text" ng-if="question_text.text"></span>
+
+            <img ng-src="{{question_text.image_url}}" id="question-image" alt="{{question_text.image_alt}}" ng-if="question_text.image_position == 'below' && question_text.image_url" />
+            </div>
+
+            <a name="reflecting"></a>
+            <div role="group" aria-label="Progress Indicator">
+                <span id="reflect" ng-if="rc.status() == rc.ALL_STATUS.ANSWERED">
+                    <ol style="background:#f8f8f8;border:2px solid #ddd;margin:1em 0;padding-top:.3em;">
+                        <li style="display:inline-block;width:30%;text-align:center;color:#414141;padding-top:.3em"><span class="fa fa-check-circle" aria-hidden="true" style="font-size:1.4em;"></span><br><span translate>Answer</span><span class="sr" translate>, Completed</span></li>
+                            <li class="fa fa-angle-double-right" style="color:#ddd;vertical-align:15%;font-size:2.2em" aria-hidden="true"></li>
+                        <li style="display:inline-block;width:30%;text-align:center;font-weight:bold;color:#111111;padding-top:.3em;"><span class="fa fa-arrow-circle-down" aria-hidden="true" style="font-size:1.4em;"></span><br><span translate>Reflection</span><span class="sr" translate>, In Progress</span></li>
+                            <li class="fa fa-angle-double-right" style="color:#ddd;vertical-align:15%;font-size:2.2em" aria-hidden="true"></li>
+                        <li style="display:inline-block;width:30%;text-align:center;color:#414141"><span class="fa fa-circle-o" aria-hidden="true" style="font-size:1.4em;"></span><br><span translate>Results</span><span class="sr" translate>, Upcoming</span></li>
+                    </ol>
+                </span>
+            </div>
+
+            <div class="others-responses" id="others-responses" data-ng-if="rc.status() == rc.ALL_STATUS.ANSWERED" tabindex="-1" role="region" aria-labelledby="ubcpi-other-answers-heading">
+
+                <span id="ubcpi-other-answers-heading" style="color:#414141" translate>Step 2) Read Other Students Answers</span>
+                <p class="ubcpi-other-answers-instructions" translate>These are samples of other student answers for this question. Read them and then compare with your answer below.</p>
+
+                <ul class="ubcpi-other-answers">
+                    <li data-ng-repeat="answer in rc.other_answers.answers">
+                        <img
+                        ng-src="{{'options[answer.option].image_url'}}"
```

Comment:
By the way, I surrounded the options[answer.option].image_url in the single quotes because otherwise I got a "could not parse remainder on [answer.option].image_url". And the same goes for many other lines